courses/models.py

```diff
@@ -658,6 +659,23 @@ def save(
         )
 
 
+def limit_to_certificate_pages():
+    """
+    A callable for the limit_choices_to param in the FKs for certificate pages
+    to limit the choices to certificate pages, rather than every page in the
+    CMS.
+    """
+    content_type = ContentType.objects.filter(
+        app_label="cms", model="certificatepage"
+    ).get()
+
+    avaialble_revisions = Page.objects.filter(content_type=content_type).values_list(
+        "id", flat=True
+    )
+
```

Comment:
How about we directly get the certificate pages list from `cms.CertificatePage` without the intervention of content type?